README.md

```diff
@@ -1,115 +1,105 @@
 
 # Thumbnailer
 
-Thumbnailer takes timestamped images as an input and produces an animation in WebP format that best fits under a given size constraint.
+Thumbnailer takes timestamped images as input and produces a WebP animation that fits a given size constraint.
 
-## Prerequisites
+## Building
 
-- [Bazel Build System](https://docs.bazel.build/versions/master/bazel-overview.html)
-- [Clang](https://clang.llvm.org/)
+#### Prerequisites
 
-On Linux, the following would install all the dependencies you need for building the API:
-```
-sudo apt-get install bazel clang unzip zip
```

Comment:
It seemed convenient to me, you prefer to remove it so that the README is shorter?

Reply:
It does not work on my Ubuntu installation. There are more steps to it [here](https://docs.bazel.build/versions/master/install-ubuntu.html). The CI might be pre-configured, so that's why we use that command it in the CI.